### PR TITLE
Hide Anatomogram with incomplete data

### PR DESCRIPTION
**Pull request by haideriqbal**
For experiments with incomplete anatomogram data, they should not display anatomograms. See issue here:

https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/127

### CODE REVIEW

**Review by lingyun1010 — CHANGES_REQUESTED** [review]
I have tested it locally, and the `anatomogram` props is required in the front-end router, so I think you are right that we need to pass an empty value for `anatomogram` props. Otherwise, the result page will not load. Please take care of the test correspondingly, instead of missing `anatomogram` but has an empty value for those exception experiments.

**Review by lingyun1010 — APPROVED** [review]
LGTM

File: src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java [hunk 1]
```diff
@@ -70,11 +82,16 @@ public JsonObject getTsnePlotData(String experimentAccession) {
...
+        if (EXPERIMENTS_WITH_NO_ANATOMOGRAM.contains(experimentAccession)) {
+            result.add(
+                    "anatomogram", GSON.toJsonTree(""));
+        } else {
+            result.add(
+                    "anatomogram",
+                    GSON.toJsonTree(
+                            ontologyAccessionsSearchService.searchAvailableAnnotationsForOrganAnatomogram(
+                                    experimentAccession)));
+        }
```
Comment (alfonsomunozpomer) [inline src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java:94]: Remember that you can rewrite this as:
```java
result.add(
        "anatomogram",
        GSON.toJsonTree(EXPERIMENTS_WITH_NO_ANATOMOGRAM.contains(experimentAccession) ?
                "" :
                ontologyAccessionsSearchService.searchAvailableAnnotationsForOrganAnatomogram(experimentAccession));
```

Now, I’m not saying that you should change it, but consider it before merging and write it however it feels more expressive and easier to understand to you.

Comment (haideriqbal) [reply src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java]: Sounds good Alfonso. I've changed it according to the suggested style. It looks much cleaner and also avoids code duplication.

**Pull request merged**